src/iarc7_motion/iarc_tasks/task_utilities/translate_stop_planner.py

```diff
@@ -53,6 +53,10 @@ def get_xyz_hold_response(self):
 
     def reinitialize_translation_stop_planner(self, x=None, y=None, z=None):
         with self._lock:
+            if (x is None or y is None or z is None):
+                rospy.logerr('Exception: Undefined behavoir when retinitializing translate stop planner')
```

Comment:
I was thinking of if you do not change the value (i.e. you pass in `None` for x or leave the default value), then the function will retain the current end goal for the respective dimension (x in my example).

Reply:
Ok so don't raise an error?

Reply:
No, but merely check each value and see if its None. If it is, then don't change the goal for that value in the planner.